src/decorators/decoratable.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Nectary;
+
+/**
+ * Objects that wish to return a
+ * default decorate should implement
+ * this interface
+ */
+interface Decoratable {
```

Comment:
@rraub Using the term "Decoratable" instead of "Decorator_Interface". I think that makes it more clear what the distinction between "Decorator" and "Decoratable" is.

Let me know what you think.